ext/ctype/ctype.stubs.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/** @return bool */
+function ctype_alnum(string $text) {}
```

Comment:
Shouldn't all of those have be ``function_name(string $text): bool {}`` ?

Reply:
I'm not sure about it. I'm based on other stubs that don't have, like: https://github.com/php/php-src/blob/f6f7afdc050a6546d147d0b0d6cf6a9a169be8e3/ext/exif/exif.stub.php

But I'll try this out now

Reply:
If you compare the generated arginfo and the original ones you see the boolean return type is missing. Some of the other ones can't do that because they return a union of type, in the EXIF case ``array|false`` which currently can't be represented in PHP's type system.

Reply:
Makes sense, I'll try now

Reply:
Not at all

Reply:
Well this did add the return type on the first one however no idea why the other ones are not getting generated.